code/game/objects/items/eightball.dm

```diff
@@ -41,6 +41,13 @@
 		new /obj/item/toy/eightball/haunted(get_turf(src))
 		qdel(src)
 
+/obj/item/toy/eightball/examine(mob/user)
+	..()
+	if(on_cooldown)
```

Comment:
Hmmm.... @lzimann how might I go about telling it to print the time left [see line 69 I just don't know how to convert that into a number I can print]